config/initializers/devise.rb

```diff
@@ -6,7 +6,7 @@
   config.env_key = lambda do |env|
     if env['REMOTE_USER']
       env['REMOTE_USER']
-    elsif Rails.env.development? && ENV['REMOTE_USER']
+    elsif Rails.env.development? && ENV.fetch('REMOTE_USER')
```

Comment:
Should this be `ENV.fetch('REMOTE_USER', nil)` otherwise this could raise a `KeyError` here.

Reply:
ok, thanks!